jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/OpenShellInContainerDialog.kt

```diff
@@ -98,7 +99,8 @@ class OpenShellInContainerDialog(
             }
             EcsTelemetry.runExecuteCommand(project, Result.Succeeded, EcsExecuteCommandType.Shell)
         } catch (e: Exception) {
-            LOG.error(e) { "Failed to start interactive shell" }
+            e.notifyError(message("ecs.execute_command_failed"))
+            LOG.warn(e) { "Failed to start interactive shell" }
```

Comment:
Why lower the severity? Error seems justified.

Reply:
Errors get reported as an IDE error and shown to users with option to disable plugin.

Reply:
Interesting